src/docfx/build/redirection/RedirectionProvider.cs

```diff
@@ -178,8 +188,26 @@ private static RedirectionItem[] LoadRedirectionModel(ErrorBuilder errors, strin
                     // Rebase source_path based on redirection definition file path
                     var basedir = Path.GetDirectoryName(fullPath) ?? "";
 
-                    var results = new List<RedirectionItem>();
-                    foreach (var item in redirections.Concat(renames))
+                    var allRedirections = redirections.Concat(renames);
+
+                    if (isSubRedirectionFile && repository != null)
```

Comment:
Why the difference here? `source_path` is defined as a path relative to the redirection file itself, which is also the assumption before the change.

Reply:
As we discussed, all path for redirection rules would be converted to ones that are relative to repo root.